packages/@expo/fingerprint/CHANGELOG.md

```diff
@@ -4,10 +4,13 @@
 
 ### 🛠 Breaking changes
 
+- This version includes fingerprint result breaking changes. ([#31638](https://github.com/expo/expo/pull/31638) by [@kudo](https://github.com/kudo))
```

Comment:
I wonder if there's a way to indicate that every version bump is a breaking change. Might be simpler for this library.

Reply:
so far it's still maintainable and i still want to keep patch version bump as non-breaking change. having the changelog helps the person doing package publish to understand whether it should be a major/minor/patch version bump.
if maintaining breaking changes comes much hassle in the future, we could clarify a better approach for this. thanks!